opal/scaffolding/plugin_scaffold/app/plugin.py.jinja2

```diff
@@ -13,9 +15,9 @@ class {{ name.title() }}Plugin(plugins.OpalPlugin):
     javascripts = {
         # Add your javascripts here!
         'opal.{{ name }}': [
-            # 'js/{{ name }}/app.js',
```

Comment:
Is this right? I might be wrong but I think this should stay like it was

Reply:
(on mobile) Might be a preprocessor/concatenator argument...

Reply:
for context it get's passed to https://github.com/openhealthcare/opal/blob/v0.10.0/opal/templates/plugins/javascripts.html#L3

Reply:
These strings end up passed to 
```
# ./opal/templates/plugins/javascripts.html
<script type="text/javascript" src="{% static variable %}"></script>
```
Does `{% static %}` take url paths or file paths ? I've never had to know... 
Looks like it's a URI to me?

Reply:
Fixed in https://github.com/openhealthcare/opal/commit/97d2b94672c3b959e2d2dd4941c66e99825dee19 :)